tox.ini

```diff
@@ -45,6 +45,7 @@ commands =
     unit: pytest --maxfail=10 -v --cov=celery --cov-report=xml --cov-report term {posargs}
     integration: pytest -xsv t/integration {posargs}
 setenv =
+    PIP_EXTRA_INDEX_URL=https://graingert.co.uk/celery-wheelhouse/repo/simple/
```

Comment:
also happy to take other names on this. "hamster-wheels" ?

Reply:
I guess being explicit is better so I like the original name.

Reply:
```suggestion
    PIP_EXTRA_INDEX_URL=https://celery.github.io/celery-wheelhouse/repo/simple/
```